korangar/src/graphics/instruction.rs

```diff
@@ -177,8 +177,15 @@ pub struct EntityInstruction {
     pub world: Matrix4<f32>,
     pub texture_position: Vector2<f32>,
     pub texture_size: Vector2<f32>,
+    pub texture_top_left: Vector2<f32>,
+    pub texture_bottom_left: Vector2<f32>,
+    pub texture_top_right: Vector2<f32>,
+    pub texture_bottom_right: Vector2<f32>,
+    pub depth_extra: f32,
```

Comment:
What is depth_extra? Can't merge in into depth_offset instead? (in linear space of course)

Reply:
Yeah, i merged with depth_offset, it worked.